.changes/next-release/Bug Fix-488470d7-db3a-48fe-8c06-2b499e374995.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Make maven command less error prone; delete temp files"
```

this changelog description is not meaningful to anyone outside of this PR. it needs, at minimum, a prefix indicating which feature it's referring to. 

please review https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog

```suggestion
	"description": "Amazon Q Transform: Make maven command less error prone; delete temp files"
```